tests/Extensions/Hosting.Services.Web.UnitTests/ResponseHeadersMiddlewareTests.cs

```diff
@@ -29,9 +29,9 @@ public async Task InvokeAsync_WithoutActivity_NotFailing()
 			ResponseFeatureForCallbacks feature = new ResponseFeatureForCallbacks();
 			HttpContext context = new DefaultHttpContext();
 			context.Features.Set<IHttpResponseFeature>(feature);
-			ResponseHeadersMiddleware middleware = new ResponseHeadersMiddleware(mock.Object);
+			IMiddleware middleware = new ResponseHeadersMiddleware(mock.Object);
 
-			await middleware.InvokeAsync(context);
+			await middleware.InvokeAsync(context, s => Task.CompletedTask);
```

Comment:
Same, configureawait? Although here you give it a completed task here so probably doesn't make any difference.

Reply:
Added, thanks